packages/graphql/src/services/sessions.ts

```diff
@@ -17,38 +18,19 @@ import { GraphQLError } from 'graphql'
 import { max, mean, median, min, quantileSeq, std } from 'mathjs'
 import schedule from 'node-schedule'
 import { ISession } from 'src/schema/session'
+import {
+  AllQuestionInstanceTypeData,
+  AllQuestionTypeData,
+  QuestionResults,
+  QuestionResultsChoices,
+} from 'src/types/app'
 import { sendTeamsNotifications } from '../lib/util'
 
 const scheduledJobs: Record<string, any> = {}
 
-export function processQuestionData(question: Question): AllQuestionTypeData {
```

Comment:
removed, as it was only used because of manual typing. if we want to "pick" or do anything else, this would need to be reintroduced in a betterway.

Reply:
Should we maybe re-introduce this directly with this branch in an improved fashion so that we do not need to search later on where the processQuestionData has been used previously and possibly miss an occurrence?